service/status-report/main.go

```diff
@@ -215,6 +222,25 @@ func setClusterInformation(ctx context.Context, cl client.Client, status interfa
 
 }
 
+func setStorageQuotaUtilizationRatio(ctx context.Context, cl client.Client, status interfaces.StorageClientStatus) {
+	clusterResourceQuota := &quotav1.ClusterResourceQuota{}
+	clusterResourceQuota.Name = status.GetClientName()
+
+	if err := cl.Get(ctx, client.ObjectKeyFromObject(clusterResourceQuota), clusterResourceQuota); err != nil {
+		// No need to check for NotFound because unlimited quota client will not have CRQ resource
+		if !errors.IsNotFound(err) {
```

Comment:
```suggestion
	if err := cl.Get(ctx, client.ObjectKeyFromObject(clusterResourceQuota), clusterResourceQuota); client.IgnoreNotFound(err) != nil {
```

Reply:
Pls add the comment, change I suggested is only to make a point that it can be condensed.